concordium-std/src/prims.rs

```diff
@@ -279,6 +279,10 @@ extern "C" {
         line: u32,
         column: u32,
     );
+
+    #[cfg(all(feature = "wasm-test", feature = "concordium-quickcheck", target_arch = "wasm32"))]
+    /// Generating a random number for randomised testing
```

Comment:
Please add that this is only allowed in tests. Not in smart contracts deployed to the chain.

Reply:
Done.